doc/apiary/v2/fiware-ngsi-simple.apib

```diff
@@ -89,54 +203,6 @@ Detailed description
          }
         ]
 
-
-### Entity Pagination [GET /v2/entities{?limit,offset,count}]
```

Comment:
I understand that for the "refence API blueprint" each operation (i.e. URL path) should be included one time as much. However, these cases makes sense for the "examples API blueprint". So probably it is clever to "branch" the .apib document now, in the following way:

* fiware-ngsiv2-example -> with the content of fiware-ngsi-simple.apib at develop branch at this moment (i.e. before introducing the removals in this PR)
* fiware-ngsiv2-reference -> renaming fiware-ngsi-simple.apib at this PR (the renaming will be an additional commit, to be squashed after LGTM).

@jmcanterafonseca what do you think?